### ISSUE TITLE
Not Working - Windows 10 - Gimp 2.8.16

### ISSUE THREAD

**Issue by praveenbm5** [jedireza/gimp-hidpi#7]
All icons appear black.

I copied the theme to C:\Users\Username\\.gimp-2.8\themes and changed the preferences to use this theme.

Screenshot - 

![gimp-hidpi](https://cloud.githubusercontent.com/assets/507203/21579140/c725ebe0-cfc5-11e6-8747-1c2716af5944.PNG)

**Comment by jedireza** [comment]
Thanks for opening an issue. Looks like there's an issue with paths on Windows. A PR is welcomed.

**Comment by ghost** [comment]
I don't know much about theming Gimp but noticed that all the themes I had were using png images and this one uses svg. I copied one of the png's from another theme into the image folder of this one and edited the imagerc and that image showed up with all the others still being blank. That was on Windows 10; however, on Ubuntu 16.04, this theme works as is.

**Comment by bartvanandel** [comment]
Yes this is due to missing support for vector icons in the Windows build of GIMP. According to the [2.9.4 Release Notes](https://www.gimp.org/news/2016/07/13/gimp-2-9-4-released/) (I know, different version):

> Vector icons are an experimental feature, available after using the `--enable-vector-icons` build configure option.

I don't know whether it is possible to extract the build configuration options from the executable (`gimp-console-2.8.exe --version --verbose` doesn't do the job), but I guess this option either was not enabled during build, or is not supported for Windows at this moment. _Note that I've never created a Windows build of Gimp, so I'm not going to fix Gimp._

As a workaround, I've batch-converted the .svg files to .png and modified the `imagerc` file accordingly by globally replacing ".svg" with ".png". Batch conversion is pretty easy when using the [svg2png](https://www.npmjs.com/package/svg2png) npm package. When [npm](https://www.npmjs.com/) is already installed, you can do this from a Windows command prompt (adapt path to your needs):

```
npm i -g svg2png
cd "%HOMEPATH%\.gimp-2.8\themes\gimp-hidpi\images"
for %f in (*.svg) do svg2png -w 32 -h 32 -o %~nf.png %f
```

**Comment by jedireza** [comment]
Thanks for adding these details @bartvanandel

**Comment by johndevedu** [comment]
Thanks @bartvanandel! Mine were a bit too small, so I changed a couple of the parameters like this:
`for %f in (*.svg) do svg2png -w 64 -h 64 -o %~nf.png %f`

Basically, changed the 32 to 64 for width and height.

**Comment by tobia** [comment]
If you don't have Node installed, you can use Inkscape:

    cd .gimp-2.8\themes\gimp-hidpi\images
    for %f in (*.svg) do @"\Program Files\Inkscape\inkscape.com" -f %f -e %~nf.png -w 32 -h 32 -y 0.0

Adjust the size (32) to fit your display resolution. 

Remember to open `imagerc` and replace all `.svg` with `.png`
